src/components/pages/MentorMenteeMatching/MentorMenteeMatching.js

```diff
@@ -29,55 +78,32 @@ const MentorMenteeMatching = () => {
       ),
     },
     { title: 'Stack', dataIndex: 'stack', key: 'stack' },
+    {
+      title: '',
+      dataIndex: 'name',
+      key: 'assignButton',
+      render: (text, record) => (
+        <Button size="large" type="primary" onClick={() => showModal(record)}>
```

Comment:
I like the new format of your table as it is similar to the implementation of the _Pending Applications_ page for Admins which contains a primary button to open the associated modal. This keeps our UX consistent across features.